roles/core/repositories_client/tasks/centos_7.yml

```diff
@@ -10,42 +14,11 @@
     value: '0'
     no_extra_spaces: yes
   loop: [ 'base', 'updates', 'extras' ]
-  when: ( 'os' in repositories ) or ( item.name is defined and item.name == 'os' )
+  when: 
+    - ( 'os' in repositories ) or ( item.name is defined and item.name == 'os' )
+    - repositories_client_disable_distro_repos == true
   notify: command █ yum clean metadata
 
-- name: set_fact ░ Set baseurl prefix
-  set_fact:
-    baseurl_prefix: "http://{{ networks[j2_node_main_network]['services_ip']['repository_ip'] }}/repositories/$environment/{{ equipment_profile['operating_system']['distribution'] }}/$releasever/$basearch/"
-
-- name: template █ Generate /etc/yum/vars/releasever to force $releasever
-  template:
-    src: releasever.j2
-    dest: /etc/yum/vars/releasever
-    owner: root
-    group: root
-    mode: 0644
-  when: equipment_profile.operating_system.repositories_releasever is defined
-  tags:
-    - template
-
-- name: file █ Cleanup $releasever
-  file:
-    path: /etc/yum/vars/releasever
-    state: absent
-  when: equipment_profile.operating_system.repositories_releasever is not defined
-  tags:
-    - template
-
-- name: template █ Generate /etc/yum/vars/environment to set $environment
-  template:
-    src: environment.j2
```

Comment:
Also remove roles/core/repositories_client/templates/environment.j2

Reply:
You are right, I forgot to remove them.